src/localconf.js

```diff
@@ -390,6 +393,13 @@ function buildConfig(opts) {
             image: '/src/img/analysis-icons/gwas.svg',
           };
           break;
+        case 'GWASUIMockup':
```

Comment:
check with PM if we still wont to call this `GWASUIMockup` since it is not a mockup any more

also check with PM if we want to keep both this and the old `GWASApp`, or use this to replace that one

Reply:
PM agreed on renaming. keeping GWASApp code for now as reference for handling job submission when engine solution is in place. will fully replace in next iteration